src/macros/seq_ids.rs

```diff
@@ -26,22 +26,37 @@
 /// ```
 #[macro_export]
 macro_rules! seq_ids {
+	() => {};
+
+	($val:expr,) => {};
+
 	(
+		$(#[$comment:meta])*
 		$name:ident = $val:expr;
-		$( $others:ident )*
+		$($rest:tt)*

```

Comment:
Why rename the parameter from `$others` to `$rest`?

Reply:
Sorry, I was testing it out side-by-side with existing code and overwritten it with `$rest`.
Will change to `$others` if that's preferable.

Reply:
No problem, thank you.